llarp/constants/path.hpp

```diff
@@ -16,15 +17,14 @@ namespace llarp
     /// pad messages to the nearest this many bytes
     constexpr std::size_t pad_size = 128;
     /// default path lifetime in ms
-    constexpr llarp_time_t default_lifetime = 10 * 60 * 1000;
+    static constexpr auto default_lifetime = 10min;
     /// after this many ms a path build times out
-    constexpr llarp_time_t build_timeout = 30000;
+    static constexpr auto build_timeout = 30s;
 
     /// measure latency every this interval ms
-    constexpr llarp_time_t latency_interval = 5000;
-
+    static constexpr auto latency_interval = 5s;
     /// if a path is inactive for this amount of time it's dead
-    constexpr llarp_time_t alive_timeout = 60000;
+    static constexpr auto alive_timeout = 30s;
   }  // namespace path
```

Comment:
Another logic change here. (I'm just taking notes at this point, this seems like a reasonable change)

Reply:
60s is very long